tests/integration/test_fesom_2p6_pimesh_esm_tools.py

```diff
@@ -27,7 +28,7 @@ def test_process_progressive_pipeline(
         rule["pipelines"] = ["default"]
     cfg["pipelines"].append({"name": "default", "steps": []})
     pipeline = cfg["pipelines"][0]
-    pipeline["steps"] = steps
+    pipeline["steps"][:] = steps
```

Comment:
I guess this did not make a difference, so I would prefer to keep the original version:
```suggestion
    pipeline["steps"] = steps
```